modules/cloudsql-instance/variables.tf

```diff
@@ -14,6 +14,15 @@
  * limitations under the License.
  */
 
+variable "allocated_ip_ranges" {
+  description = "(Optional)The name of the allocated ip range for the private ip CloudSQL instance. For example: \"google-managed-services-default\". If set, the instance ip will be created in the allocated range. The range name must comply with RFC 1035. Specifically, the name must be 1-63 characters long and match the regular expression a-z?."
+  type = object({
+    primary = optional(string)
+    replica = optional(string)
+  })
+  default = null
```

Comment:
The default should probably be `{}` and the variable should have `nullable = false`, or you run the risk of `primary` and `replica` members not being set. Can you also remove the extra newline at line 24?

Reply:
Thanks @ludoo  - I have updated the variable definition. This will also fixed the broken tests, which might need someone to kick off the workflow.